### ISSUE TITLE
[Onboarding] Update sign-in flow

### ISSUE THREAD

**Issue by rachelhamlin** [status-im/status-react#8032]
### User Story
As an existing user, I want to log into my account or choose from multiple accounts, if I have them. 

### Description
_Type_: Feature

_Summary_: Update account sign-in flow according to new [onboarding designs](https://www.figma.com/file/dEIljL7UPbXgsZUA0Q4qlE5E/Onboarding?node-id=927%3A14702). 

#### Expected behavior
The two primary sign-in screens have minor UI adjustments.

![image](https://user-images.githubusercontent.com/11790366/57222440-dce8ca80-7002-11e9-8eb8-7fc1dd8d6420.png)

#### Actual behavior
![](https://user-images.githubusercontent.com/15470435/56672331-47554e80-6684-11e9-939e-f3c69b9157b8.png)![](https://user-images.githubusercontent.com/15470435/56672373-589e5b00-6684-11e9-9d00-5e155945c7c4.png)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 150.0 DAI (150.0 USD @ $1.0/DAI)  attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/status-im/status-react/8032/3129).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/status-im/status-react/8032/3129).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $85,324.80 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 12 months from now.
Please review their action plans below:


**1) [sudeepb02](https://gitcoin.co/profile/sudeepb02) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/status-im/status-react/8032/3129?mutate_worker_action=approve&worker=sudeepb02) | [reject worker](https://gitcoin.co/issue/status-im/status-react/8032/3129?mutate_worker_action=reject&worker=sudeepb02))_.**

Adjust the layout of the sign-in screens to match the expected behavior

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/status-im/status-react/8032/3129).

**Comment by StatusWrike** [comment]
➤ Rachel Hamlin commented:

Waiting to get bounty applicant approved; will then estimate date.

**Comment by gitcoinbot** [comment]
@sudeepb02 Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-react/8032/3129?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-react/8032/3129?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-react/8032/3129?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-react/8032/3129?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-react/8032/3129?snooze=100)

**Comment by sudeepb02** [comment]
Hi team,

I have taken up this issue and currently going through the technical documentation and contributing guidelines. I am a beginner in Clojure, so still figuring out most of the stuff. I have downloaded the app from Play Store, but I guess the screens mentioned in this issue were after the app was released. How do I reproduce this issue? And where else to look at? Please guide me in the right direction.

Thanks.

**Comment by vitvly** [comment]
@sudeepb02 Hi! Thank you for willing to contribute to the project! Please try to setup your dev environment as described [here](https://status.im/build_status/), and let us know how it goes.

The screens that need to be changed are displayed during app login and are located in 3 rows below the `User Opens Status` label in Figma (3 top ones starting with `Sign in`).

**Comment by hesterbruikman** [comment]
Welcome @sudeepb02 and thanks for working on this issue. For the screens, this is your best, up-to-date reference: https://www.figma.com/file/dEIljL7UPbXgsZUA0Q4qlE5E/Onboarding?node-id=1625%3A20964

@errorists , can you please confirm?

**Comment by sudeepb02** [comment]
Hi @siphiuel @hesterbruikman 

Thanks for the resources. I am done with the initial setup. I installed all the packages on a VPS. Is it possible to compile the app, and then download and install the apk to my device? I tried running `make startdev-android-real` and ran into errors.

**Comment by vitvly** [comment]
@sudeepb02 which errors exactly did you run into? Did also try with an AVD?

**Comment by sudeepb02** [comment]
Hi @siphiuel . I am running it on a VPS. So won't be able to use an AVD.

**Comment by vitvly** [comment]
@sudeepb02 Maybe you can use VNC or some other remote desktop tool? You definitely need to be able to have a proper development environment so that to iterate quickly.

**Comment by hesterbruikman** [comment]
Copy update on these screens:

On account list view: Recover a key > Access key
On numpad: Recover key > Access key
On password entry: Recover key > Access key

**Comment by gitcoinbot** [comment]
@sudeepb02 Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-react/8032/3129?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-react/8032/3129?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-react/8032/3129?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-react/8032/3129?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-react/8032/3129?snooze=100)

**Comment by sudeepb02** [comment]
I will try setting up a VNC/remote desktop and set up a proper development environment. Will post updates once done

**Comment by gitcoinbot** [comment]
@sudeepb02 Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-react/8032/3129?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-react/8032/3129?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-react/8032/3129?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-react/8032/3129?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-react/8032/3129?snooze=100)

**Comment by sudeepb02** [comment]
Hello Team,
Because of not having a proper dev setup, I'm sorry that I won't be able to complete this issue. I'd be stopping work on gitcoin so someone else can take this up.

**Comment by emmaodia** [comment]
I have applied to get this task done. Do I have to be approved first before I continue?
Thanks.

**Comment by rachelhamlin** [comment]
@emmaodia yes, let's get you approved first. :) Thanks for applying! 

@StatusSceptre could you please approve @emmaodia? 🙏

Thanks for your effort @sudeepb02.

**Comment by flexsurfer** [comment]
@rachelhamlin @StatusSceptre  as i know @siphiuel is working on it

**Comment by emmaodia** [comment]
Thank you @rachelhamlin 

@flexsurfer I think the gitcoin activity tracker is broken as I do not **see** anyone working on it and it indicates that I have started work. In the meantime, I will apply to other status projects.

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Cancelled** 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 9 months, 1 week from now.
Please review their action plans below:


**1) [xriz007](https://gitcoin.co/profile/xriz007) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/status-im/status-react/8032/3129?mutate_worker_action=approve&worker=xriz007) | [reject worker](https://gitcoin.co/issue/status-im/status-react/8032/3129?mutate_worker_action=reject&worker=xriz007))_.**

I'd create separate screen showing the multiple accounts and a verification screen for the selected account to login

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/status-im/status-react/8032/3129).

**Comment by flexsurfer** [comment]
hey @emmaodia please do not start work, core contributor is working on this issue, thanks

**Comment by rachelhamlin** [comment]
Sorry about that @emmaodia! You might be interested in this issue instead? #8130

**Comment by flexsurfer** [comment]
sorry #8130 already done

**Comment by emmaodia** [comment]
Okay @Audrey

@rachelhalim I will go through the status issues page and find the ones I can work on.

Thanks guys.

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Cancelled** 

<hr>

__The funding of 150.0 DAI  (150.0 USD @ $1.0/DAI) attached to this issue has been **cancelled** by the bounty submitter__

 
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $139,437.91 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
